tools/benchmarksAnalyzer/src/main/kotlin/main.kt

```diff
@@ -76,11 +65,28 @@ object TeamCityConnector: Connector() {
     }
 }
 
+object DBServerConnector : Connector() {
+    override val connectorPrefix = ""
+    val serverUrl = "https://kotlin-native-perf-summary.labs.jb.gg"
```

Comment:
Why are all urls hardcoded? I think it could be a property defined by TC instance. And I think for the security reasons it's better to not show the internal URL.

Reply:
>  I think it could be a property defined by TC instance. 

It can't be, this code is also compiled and used in different way. As JS it produces on our server detailed information (frontend JS can't read some local files or variables), it also is used as command line tool on developers machine and creating extra environmental variables or files is overkilling, I think.